changelogs/54140.yml

```diff
@@ -0,0 +1,4 @@
+title: "delete a not used field"
+type: fix
+description: |
+  We remove the unused field "lead" from EmailAddress.
```

Comment:
Please move this file to the right folder, ie. `changelogs/unreleased/54140.yml`